DearPyGui/src/core/AppItems/basic/mvMenuItem.cpp

```diff
@@ -23,15 +23,18 @@ namespace Marvel {
 	}
 
 	mvMenuItem::mvMenuItem(const std::string& name)
-		: mvBoolPtrBase(name, false) {}
+		: mvBoolPtrBase(name, false) 
+	{
+		m_description.disableAllowed = true;
+	}
 
 	void mvMenuItem::draw()
 	{
 		ScopedID id;
 		mvImGuiThemeScope scope(this);
 
 		// create menuitem and see if its selected
-		if (ImGui::MenuItem(m_label.c_str(), m_shortcut.c_str(), m_check ? m_value.get() : nullptr, m_core_config.enabled))
+		if (ImGui::MenuItem(m_label.c_str(), m_shortcut.c_str(), m_check ? m_value.get() : nullptr))
```

Comment:
This widget directly supports disabling in imgui itself. Which supports a few weird cases. Checkmarks for example.